README.md

```diff
@@ -1 +1,1175 @@
-# eip-message-transformation
\ No newline at end of file
+[![Build Status](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)
+
+# message-transformation---ballerina
+There are different ways of message transformation methods in EIP (Enterprise Integration Patterns). In this guide, we are focusing on 'content filter', 'claim check' and 'content enricher' message transformation methods between services using an example scenario.
```

Comment:
Change to 'There are different message transformation methods in Enterprise Integration Patterns (EIP). This guide focuses on...' .